acquia_lift_profiles/js/acquia_lift_profiles.debug.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * Defines JavaScript functions needed by the Acquia Lift debugger in
+ * order to provide customer profile information.
+ */
+
+(function ($, Drupal) {
```

Comment:
File needs to be cleaned up to match coding standards: https://www.drupal.org/node/172169